src/js/components/Wizard.js

```diff
@@ -1,9 +1,9 @@
 import React, {cloneElement} from 'react';
 import { browserHistory } from 'react-router';
 import werewolfService from '../services/werewolf';
-import gamesTypes from '../core/gameTypes';
+// import gamesTypes from '../core/gameTypes';
 
-const DEFAULT_GAME_MODE = gamesTypes.balanced;
+// const DEFAULT_GAME_MODE = gamesTypes.balanced;
```

Comment:
The wizzard.js file is used for reference only. It's going to be deleted in the redux branch.